cli/commands/authenticate.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+const authenticationTasks = require('../../lib/authentication-tasks');
+const config = require('../../config');
+const prompt = require('prompt');
+const logger = config.logger;
+
+module.exports = function(program) {
+	program.command('authenticate')
+		.description('Obtain AWS session credentials using a user profile from  ~/.aws/credentials and saves them to ' +
+			'./.env.session. This file will be overwritten every time this command is run. If you do not have ' +
```

Comment:
Scope is a little ambiguous.  Will `~/.aws/credentials` be overwritten?

(English is hard...)